camelot/view/controls/delegates/filedelegate.py

```diff
@@ -52,6 +53,7 @@ def get_standard_item(cls, locale, model_context):
         item = super(FileDelegate, cls).get_standard_item(locale, model_context)
         cls.set_item_editability(model_context, item, False)
         if model_context.value is not None:
+            item.setData(model_context.value.verbose_name, Qt.ItemDataRole.EditRole) # FIXME: bind object?
```

Comment:
ik denk dat dat op dit moment nt uitmaakt, omdat je in die editor toch niet kan 'editen' heeft die eigenlijk geen edit role nodig.  de edit is dmv een field action.

mogelijk is 'in de toekomst' een soort van bind hier wel nuttig, zodat de ui informatie heeft over 'waar de file is', en die bvb rechtstreeks uit een DMS systeem kan opvragen ...